src/Symfony/Component/Notifier/Bridge/Sendinblue/README.md

```diff
@@ -0,0 +1,26 @@
+Sendinblue Notifier
+==============
```

Comment:
Missing some `=` to match the title length

Reply:
done